system-reinstall-bootc/src/prompt.rs

```diff
@@ -9,7 +9,8 @@ fn prompt_single_user(user: &crate::users::UserKeys) -> Result<Vec<&crate::users
     let prompt = indoc::formatdoc! {
         "Found only one user ({user}) with {num_keys} SSH authorized keys.
         Would you like to import its SSH authorized keys
-        into the root user on the new bootc system?",
+        into the root user on the new bootc system?
+        This is the key that will be used to access the bootc system.",
```

Comment:
"bootc system" stutters here. I wonder if we even need to clarify "bootc" system in this context, I think it's implicitly understood. Maybe:

> Would you like to copy the SSH authorized keys for the root user on the new system? Then you can log in as `root@` using those keys.